src/main/java/com/github/markusbernhardt/seleniumlibrary/CustomRobotDriverElement.java

```diff
@@ -21,12 +22,14 @@
     public CustomRobotDriverElement() throws NoSuchFieldException, IllegalAccessException {
         try {
             CustomRobotDriverElement.s = getLibraryInstance();
-        } catch (javax.script.ScriptException e) {
-            throw new SeleniumLibraryNonFatalException("Cannot create SeleniumLibrary instance: " + e.getMessage());
+        } catch (ScriptException e) {
+            throw new SeleniumLibraryNonFatalException("Cannot create SeleniumLibrary instance: \n"
+                    + ExceptionUtils.getStackTrace(e));
```

Comment:
The code can be simplified by just providing an exception as the second parameter:

https://github.com/Hi-Fi/robotframework-seleniumlibrary-java/blob/cbe8ecb27069dfc5f24f5dd0e3e944683a37caaf/src/main/java/com/github/markusbernhardt/seleniumlibrary/SeleniumLibraryNonFatalException.java#L27-L29

Reply:
Done